### PR TITLE
Add setting to disable swiping between articles

### PR DESCRIPTION
**Pull request by NWuensche**
Fix one of the requests in #734 .

### CODE REVIEW

**Comment by di72nn** [comment]
> should be implemented like "Tap-to-scroll"

`if(!tapToScroll) return false;` at the start of the method is what I meant by that. It would be slightly better optimized and more concise (in a way).

File: app/src/main/java/fr/gaulupeau/apps/Poche/ui/preferences/SettingsActivity.java [hunk 1]
```diff
@@ -103,6 +103,9 @@ protected void onCreate(Bundle savedInstanceState) {
 
         private boolean keepScreenOnChanged;
         private boolean oldkeepScreenOn;
+
+        private boolean swipeArticlesChanged;
```
Comment (di72nn) [inline app/src/main/java/fr/gaulupeau/apps/Poche/ui/preferences/SettingsActivity.java]: All these `*Changed` flags are meant for settings that need to do some action in `applyChanges()` which is not the case for this.

File: app/src/main/res/xml/preferences.xml [hunk 2]
```diff
@@ -166,6 +166,11 @@
                 android:summary="@string/pref_desc_ui_scrollOverBottom"
                 android:inputType="number"
                 android:defaultValue="3"/>
+            <CheckBoxPreference
+                android:key="@string/pref_key_ui_swipeArticles"
+                android:title="@string/pref_name_ui_swipeArticles"
+                android:summary="@string/pref_desc_ui_swipeArticles"
+                android:defaultValue="true"/>
```
Comment (di72nn) [inline app/src/main/res/xml/preferences.xml]: I would rather have it disabled by default. The feature is not implemented well and may be confusing. But it is debatable.

**Comment by NWuensche** [comment]
I did delete the `*Changed` Stuff and used the `tapToScroll` scheme. I will also rebase the commits if you want if you approve everything until now.

File: app/src/main/java/fr/gaulupeau/apps/Poche/data/Settings.java [hunk 3]
```diff
@@ -388,6 +388,14 @@ public void setKeepScreenOn(boolean keepScreenOn) {
         setBoolean(R.string.pref_key_ui_keepScreenOn, keepScreenOn);
     }
 
+    public boolean getSwipeArticles() {
+        return getBoolean(R.string.pref_key_ui_swipeArticles, true);
```
Comment (di72nn) [inline app/src/main/java/fr/gaulupeau/apps/Poche/data/Settings.java]: That should be `false` too

**Comment by di72nn** [comment]
Sorry for nitpicking - just trying to keep the code clean and to reduce technical debt. Thank you for your continued efforts!

**Comment by NWuensche** [comment]
No problem, I'm glad that you're helping me! I changed the `true` now. Is everything OK like this? If so, I would squash everything into one commit.

**Comment by di72nn** [comment]
Yeah, looks good

**Comment by NWuensche** [comment]
Everything should be nice now.

**Pull request merged**